### ISSUE TITLE
Date input format

### ISSUE THREAD

**Issue by chrisschaub** [signalpoint/date#3]
So, the date input format shows m/d/yyyy in the widget settings, but it gets rendered as d/m/yyyy which is confusing. I think it's just because the parts just get assembled as they come in.

**Comment by signalpoint** [comment]
@chrisschaub Yeah, somewhere in date_field_widget_form() it is assembling the widget for the form. Doing a dpm() on field/instance will hopefully reveal the ordering of the widgets. Then the assembly of the individual inputs would need to be updated to understand the ordering/weights.
